### ISSUE TITLE
Questions on the Tables / Figures of the paper for comparisons

### ISSUE THREAD

**Issue by ThibaultGROUEIX** [devlearning-gt/3DShapeGen#11]
Hi,

Thanks again for this work and the awesome code / data release! We will be trying to compare with your tables so here a few queries to make sure we are doing things properly.

Our main source of hesitations comes from :
* Does SDFNet denotes a method taking estimated depth as input or GT?
* Throughout the paper, which version of Table's 6 background is used?

Here is my current understanding please correct me if you spot something wrong : 

- Fig 1 : SDFNet trained with *_**estimated**_* depth
- Fig 2 : SDFNet trained with ***estimated*** depth 
- Fig 3 : SDFNet trained with ***ground-truth*** depth 

- Table 2 : GenRe data. 3 categories for training / 10 for testing , using estimated depth to train/test
- Table 3 : ShapeNet data. 13 categories for training / 42 for testing , renderings with 2 DoF, using estimated depth to train/test
- Table 4 : ShapeNet data. 13 categories for training / 42 for testing , renderings with 3 DoF, using estimated depth to train/test
- Table 5 : ShapeNet data. 13 categories for training / 42 for testing ,  using ground-truth depth to train/test
- Table 6 : ShapeNet data. 13 categories for training and testing ,  using estimated and ground-truth depth to train/test

Some questions to make sure I understand : 
* Why doesn't SDFNet Orcl results in Table 6  appear in Table 5?
* Why isn't SDFNet Est results in Table 6 the same as SDFNet  Est - Seen Classes in  Table 5?

Thank you so much for your precious help,
Thibault

**Comment by ngailapdi** [comment]
Hi Thibault,

Thank you for your interest in our work. 
`Does SDFNet denotes a method taking estimated depth as input or GT?`
In the paper, in sections 4.5 (table 5, figure 6) and 4.7 (table 7, figure 8) the models are trained entirely with GT 2.5D data. For the rest of the tables/figures, SDFNet/SDFNet Est means SDFNet with estimated 2.5D sketches. 

`Throughout the paper, which version of Table's 6 background is used?`
We used LRBg rendering condition throughout the paper.

`Table 2 : GenRe data. 3 categories for training / 10 for testing , using estimated depth to train/test
  Table 3 : ShapeNet data. 13 categories for training / 42 for testing , renderings with 2 DoF, using estimated depth to train/test
  Table 4 : ShapeNet data. 13 categories for training / 42 for testing , renderings with 3 DoF, using estimated depth to train/test
  Table 5 : ShapeNet data. 13 categories for training / 42 for testing , using ground-truth depth to train/test
  Table 6 : ShapeNet data. 13 categories for training and testing , using estimated and ground-truth depth to train/test
`
This looks correct to me. 

`Why doesn't SDFNet Orcl results in Table 6 appear in Table 5?`
Thank you for spotting this ambiguity. In table 6, we evaluated on a smaller scale (100 objects per category) while in table 5 we evaluated on all objects 
 
`Why isn't SDFNet Est results in Table 6 the same as SDFNet Est - Seen Classes in Table 5?`
In table 5, we used ground truth 2.5D sketches to train and evaluate. I think you probably meant table 4. The main difference between table 4 and 6 is that in table 4, the models were trained with data rendered with background and various lighting conditions while in table 6, we trained the models on the "basic" rendering condition and test on "basic" (first column) and LRBg rendering (second column) conditions.
